# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/token/hotfix-includedir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- token-24.09/cmake/tokenPreInstall.cmake.vanilla	2026-09-27 20:30:49.185105425 +0300
+++ token-24.09/cmake/tokenPreInstall.cmake	2026-09-27 20:32:28.321982034 +0300
@@ -5,4 +5,4 @@
   @ONLY)
 
 install (FILES ${PROJECT_BINARY_DIR}/${PROJECT_NAME}/Options.h
-  DESTINATION include/${PROJECT_NAME}/${token_VERSION}/token)
+  DESTINATION include/token)
--- token-24.09/cmake/tokenMacros.cmake.vanilla	2026-09-27 20:31:44.252336751 +0300
+++ token-24.09/cmake/tokenMacros.cmake	2026-09-27 20:32:09.206180475 +0300
@@ -36,7 +36,7 @@
 function(token_export_header target file)
   token_get_kit_name(name dir_prefix)
   generate_export_header(${target} EXPORT_FILE_NAME ${file})
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${file}  DESTINATION include/${PROJECT_NAME}/${token_VERSION}/${dir_prefix})
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${file}  DESTINATION include/${dir_prefix})
 endfunction()
 
 # Builds a source file and an executable that does nothing other than
--- token-24.09/token/CMakeLists.txt.vanilla	2026-09-27 20:42:06.016737044 +0300
+++ token-24.09/token/CMakeLists.txt	2026-09-27 20:43:10.470919353 +0300
@@ -25,7 +25,7 @@
   list(APPEND headers "${class}.h")
   install(
     FILES "${class}.h"
-    DESTINATION "include/token/${token_VERSION}/token/${headerDir}"
+    DESTINATION "include/token/${headerDir}"
   )
   if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${class}.cxx")
     list(APPEND sources "${class}.cxx")
@@ -47,13 +47,13 @@
   PUBLIC
     $<BUILD_INTERFACE:${token_SOURCE_DIR}>
     $<BUILD_INTERFACE:${token_BINARY_DIR}>
-    $<INSTALL_INTERFACE:include/token/${token_VERSION}>
+    $<INSTALL_INTERFACE:include/token>
 )
 token_install_library(token)
 generate_export_header(token EXPORT_FILE_NAME Exports.h)
 install(
   FILES ${CMAKE_CURRENT_BINARY_DIR}/Exports.h
-  DESTINATION include/token/${token_VERSION}/token
+  DESTINATION include/token
 )
 
 add_executable(tokenize
